### ISSUE TITLE
Tab with fields also appears in the field group details

### ISSUE THREAD

**Issue by hugofmsilva** [joomla-projects/custom-fields#112]
#### Steps to reproduce the issue
1. Create field group
2. Create all needed fields and assign each one to previously created field group
3. After that, when i edit the field group, i see one extra tab with the fields and all inputs like in articles
#### Expected result

The tab with extra fields only need to appear in articles in categories with additional fields assigned and not in field group details.
#### Actual result

Currently,the tab with extra fields appear in articles in categories with additional fields assigned and  in the field group details.
#### System information (as much as possible)

Last git version.
#### Additional comments

Sorry my english.
![attachment](https://cloud.githubusercontent.com/assets/4529320/15943634/cf900b0c-2e81-11e6-913a-724102556b25.png)

**Comment by laoneo** [comment]
Thanks for the bug report, fix is coming soon.

**Comment by coolcat-creations** [comment]
It´s hard to follow if everything is closed. Don´t we proceed like for Joomla! Core with two tests and close the issue then?

**Comment by laoneo** [comment]
I know, but the lack of interest of people willing to test or provide feedback, except you doesn't allow to set up such a process. Making progress with two test is hard. We also don't have a issues backend as issues.joomla.org is.
